AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -579,7 +580,6 @@ - (void)openURLInAuthenticationSessionWith:(NSURL *)url API_AVAILABLE(ios(11)) {
     if (!strongSelf) {
       return;
```

Comment:
We should still end the bg task before returning here.

Reply:
Fixed